mephisto/client/review/review_server.py

```diff
@@ -137,8 +210,26 @@ def after_request(response):
         response.headers.add("Cache-Control", "no-store")
         return response
 
-    thread = threading.Thread(target=consume_data)
-    thread.start()
+    if all_data:
+        # if reading all data points, all data is loaded into memory before the app starts
+        if database_task_name is not None:
+            data_source = mephistoDBReader()
+        elif json:
+            data_source = json_reader(iter(sys.stdin.readline, ""))
+        else:
+            data_source = csv.reader(iter(sys.stdin.readline, ""))
+            if csv_headers:
+                next(data_source)
+
+        all_data_list = []
+
+        for row in data_source:
+            all_data_list.append(row)
```

Comment:
maybe I'm missing something, but the above 3 lines here could just be `all_data_list = list(data_source)` no?

Reply:
You are correct, I am still getting accustomed to advanced python syntax haha. Will change